terraform/10-video-hearing-wowza/modules/wowza/network.tf

```diff
@@ -3,6 +3,8 @@ data "azurerm_client_config" "current" {
 
 locals {
   dns_zone_name = var.environment == "prod" ? "platform.hmcts.net" : "sandbox.platform.hmcts.net"
+  ip_list       = [for vm in azurerm_linux_virtual_machine.wowza : vm.private_ip_address]
+  ip_csv        = join(",", local.ip_list)
 }
 
```

Comment:
I would actually probably re-do this entire ruleset to be honest as its not in line with best practices.

1. I would have a rule with a of Priority 4096 which is DenyAllVnetInbound, so blocks all Vnet Traffic other than the rules with a lower priority. 
2. I would then keep Azure-LB-Probe, but add 8087 & 443 to the ports.
3. I would then amend or merge the 'REST' & 'RTMPS' rule to tighten up security, as they currently allow connection from ANY, I would only allow from specific IPs/Subnets/ServiceTags required.